src/_pluggy/__init__.py

```diff
@@ -0,0 +1,26 @@
+#
```

Comment:
Why would the module be named `_pluggy`?  something like `pluggedcode` `plugincode` would be better IMHO